domain/src/main/java/org/oppia/domain/oppialogger/OppiaLogUploader.kt

```diff
@@ -0,0 +1,66 @@
+package org.oppia.domain.oppialogger
+
+import android.app.Application
+import androidx.work.Constraints
+import androidx.work.Data
+import androidx.work.ExistingPeriodicWorkPolicy
+import androidx.work.NetworkType
+import androidx.work.PeriodicWorkRequest
+import androidx.work.WorkManager
+import java.util.concurrent.TimeUnit
+
+private const val OPPIA_EXCEPTION_WORK = "OPPIA_EXCEPTION_WORK_REQUEST"
+private const val OPPIA_EVENT_WORK = "OPPIA_EVENT_WORK_REQUEST"
+
+class OppiaLogUploader() {
```

Comment:
I am a bit unsure what the purpose of this class is. This is not an uploader since its not doing any uploading. It looks like its only creating the worker requests right? Can we rename this class accordingly?

Reply:
Aah, I named it so because the work requests are after all the ones that will start the process of uploading. Changed it to `OppiaLogUploadWorkRequest`, is that fine now ?